### ISSUE TITLE
Feature request: ability to share design using the URL

### ISSUE THREAD

**Issue by blakek** [honzaap/Systemizer#56]
First, thanks for this tool! I've used it several times in the last month to help think through things.

Currently, you can send someone an exported file and they can import it. I'd like to also share a system using only the URL. For example, the TypeScript Playground uses a compressed and Base64 encoded string:

https://www.typescriptlang.org/play?#code/MYewdgzgLgBAhmAogDzgWwA4BsCmA1OAJwEs4AjXGAXhgCIAJHLLEGAdREKwBMBCWgFChIIXADoWAcwAUCFOmz4ipCjgCUAmDCA

I figured this is likely the same state that's exported in "File > Save file", but encoded and placed in the URL.

**Comment by honzaap** [comment]
Yeah I though about this many many times... I'm aware that the URL you get from IFrame embedding is a mess. However I've tried many things to make it smaller but managed to get it only to this... There is no backend for this application so I guess for no the only way is using the url to edit embedded system.
